examples/chart-chart/README.md

```diff
@@ -0,0 +1 @@
+# Chart Chart
```

Comment:
Chart Deprecated

Demonstrates how to migrate from the deprecated `chartSvgCartesian` & `chartCanvasCartesian` components to the combined `chartCartesian` component.